<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enter</class>
 <widget class="QWidget" name="Enter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>87</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Логин:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Сервер:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="eLogin"/>
      </item>
      <item>
       <widget class="QLineEdit" name="ePassword">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eServerAddr">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
